src/isomorphic/children/__tests__/ReactChildren-test.js

```diff
@@ -431,13 +429,14 @@ describe('ReactChildren', function() {
     expect(console.error.calls[0].args[0]).toContain('use of a keyed object');
   });
 
+  /*eslint-disable no-unused-expressions */
```

Comment:
I like this idea. I'll grep for `/*eslint-disable no-unused-expressions */` and do that.